docs/configuration/actions.md

```diff
@@ -268,3 +268,21 @@ actions:
 
 Please note that `%changelog` expects dashes in a changelog entry, but the dash
 would be interpreted by `echo` as an option so we need to quote it.
+
+-----
+If you want to change the working directory of a command or multiple commands, don't do this, it won't work as expected:
+
+```yaml
+actions:
+  post-upstream-clone:
+    - cd /tmp
+    - touch test_file"
+\```
```

Comment:
Sorry, I had to add the backslash there for the suggestion not to end too early.